portal/gil/templates/gil/base.html

```diff
@@ -387,13 +387,14 @@ <h4 class="box-modal__title">{{_("Session Timed Out")}}</h4><p class="box-modal_
                     onwarning: function() {$("#session-warning-modal").modal("show"); if (sessMon.modalShown) sessMon.intervalMonitor = setInterval(function(){ sessMon.ontimeout() }, 2 * 60 * 1000);}
               });
               window.sessMon = sessMon;
-               // Configure the session timeout warning modal
+              var warningText = i18next.t("Your session will expired in approximately [time] seconds due to inactivity.".replace("[time]",(sessMon.timeBeforeWarning / 1000)));
```

Comment:
That should probably be "expire".

Reply:
Could you keep consistent with your other `replace()` calls and `{}` (unless you're already using `[]` everywhere)?

Reply:
Good idea moving this into the frontend by the way! :)